src/js/toolbar.js

```diff
@@ -356,6 +356,7 @@ var Toolbar;
         showAndUpdateToolbar: function () {
             this.modifySelection();
             this.setToolbarButtonStates();
+            this.base.trigger('updateToolbar', {}, this.base.getFocusedElement());
```

Comment:
Can this go either at the beginning or the end of the logic (before `this.modifySelection()` or after `this.setToolbarPositon()`)?